### PR TITLE
feat(unlock-app) - header component & export component

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
header component 

Screenshot 2022-12-12 at 13 50 04

Screenshot 2022-12-12 at 13 50 11

Screenshot 2022-12-12 at 13 50 16

Screenshot 2022-12-09 at 21 13 17

Screenshot 2022-12-09 at 21 16 29

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/ui/lib/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -165,5 +165,3 @@ const Footer = ({
     </footer>
   )
 }
-
-export default Footer
```
Comment (julien51) [inline packages/ui/lib/components/Footer/Footer.tsx:169]: Why no default export?

**Review by julien51 — COMMENTED** [review]
I am confused about that this does?

**Comment by julien51** [comment]
Please add screenshots :)

**Review by julien51 — APPROVED** [review]
LGTM

File: packages/ui/lib/components/Footer/Footer.tsx (hunk 1 above)
Comment (kalidiagne) [reply packages/ui/lib/components/Footer/Footer.tsx]: ah, restored!

**Comment by kalidiagne** [comment]
Fix to keep popover with the exact width of the menu

Screenshot 2022-12-12 at 18 27 13

Screenshot 2022-12-12 at 18 27 50

**Review by julien51 — APPROVED** [review]

**Pull request merged**